cabby/data/metagraph/utils.py

```diff
@@ -103,11 +108,88 @@ def update_osm_map(osm_map: Map,
   print("In update_osm_map: %d nodes in graph after adding." % (
     osm_map.nx_graph.number_of_nodes()))
 
+
+def add_conceptual_nodes_and_edges(graph: nx.Graph,
+                                   poi_map: Dict[Text, int],
+                                   wd_relations: pd.DataFrame):
+  for _, row in wd_relations.iterrows():
+    place_node_id = poi_map[row["place"]]
+    graph.add_edge(row["instanceLabel"], place_node_id)
+    graph.add_edge(place_node_id, row["instanceLabel"])
+
+def construct_human_readable_name(poi_row: pd.Series) -> Text:
+    name = ""
+    if isinstance(poi_row["name"], str):
+        name = poi_row["name"]
+    elif isinstance(poi_row["wikipedia"], str):
+        name = poi_row["wikipedia"][3:]
+    elif isinstance(poi_row["brand"], str):
+        name = poi_row["brand"]
+    elif isinstance(poi_row["shop"], str):
+        name = poi_row["shop"]
+    elif isinstance(poi_row["tourism"], str):
+        name = poi_row["tourism"]
+    elif isinstance(poi_row["amenity"], str):
+        name = poi_row["amenity"]
```

Comment:
Is there a chance that none of this matches? Seems like having a default like `name = 'poi'` would be prudent.

Reply:
Fixed by b0e9f5b9bd83a9fe1d5ef304d7e120501e55ce20